pkg/prometheus/promcfg.go

```diff
@@ -3052,7 +3052,25 @@ func (cg *ConfigGenerator) generateScrapeConfig(
 				})
 			}
 
-			configs[i] = cg.WithMinimumVersion("2.3.0").addFiltersToYaml(configs[i], config.Filters)
+			configs[i] = cg.addFiltersToYaml(configs[i], config.Filters)
```

Comment:
the version check is still needed, no?

Reply:
I think the reason I removed it was, I was unsure which version to check for. Is `v2.3.0` right?

Reply:
Also, I removed that there so that we can check for the version and reject the configuration at `resource_selector.go`. Maybe we need to have a discussion about this in the upcoming meeting/

Reply:
As we discussed, let me revert the logic to omit the unsupported fields. I will file an issue to discuss further.